src/com/facebook/buck/jvm/kotlin/ExternalKotlinc.java

```diff
@@ -43,7 +43,7 @@
 
 /** kotlinc implemented as a separate binary. */
 public class ExternalKotlinc implements Kotlinc, AddsToRuleKey {
-
+  @AddToRuleKey private final String kotlinCompilerVersion;
```

Comment:
Mind dropping this down with pathToKotlinc?

Reply:
dropped it down. Thanks